.ci.yml

```diff
@@ -0,0 +1,267 @@
+# Configuration for ci.nodejs.org
+#
+# `linux-x64-container-suite` list tests that are to be run in:
+#   https://ci.nodejs.org/job/rv-test-commit-linux-docker
+# using container images found at:
+#   https://github.com/rvagg/node-ci-containers
+#
+# Each test in this list needs to be defined under the `tests` list and needs:
+#  - a `label` that will be used to report to GitHub in the pull request status
+#    checks.
+#  - an `image` that corresponds to one of the images found in
+#    https://github.com/rvagg/node-ci-containers such that it can be pulled
+#    from Docker Hub as `rvagg/node-ci-containers:$image`.
+#  - an `execute` block that contains Bash that performs the build & test steps
+#    required for this build type.
+#
+# Images are assumed to contain the required packages and data needed to
+# perform the build & test steps, while the execution of those steps are
+# defined here.
+#
+# Execution is limited to the `iojs` user in each container, beginning in the
+# /home/iojs/workspace directory which maps to the current Jenkins workspace on
+# the host.
+
+linux_x64_container_suite:
+  - lint
+  - ubuntu1804
+  - ubuntu1910
+  - ubuntu1604-gcc6
+  - debian9
+  - centos7-devtoolset7
+  - centos8
+  - centos8-python2
+  - fedora30
+  - alpine310
+  - zlib
+  - openssl111
+  - withoutintl
+  - withoutssl
+  - shared
+  - debug
+  - ninja
+  - workers
+
+tests:
+  lint:
+    label: Lint JS, C++, Python, Markdown & Docs
+    image: node-linter
+    execute: |
+      make lint-py-build PYTHON=python3
+      make lint-py PYTHON=python3
+      make lint-py-build PYTHON=python2
+      make lint-ci PYTHON=python2 || {
```

Comment:
Shouldn't PYTHON be defined _before_ running `make` instead of after?

Reply:
You can do either.

```console
PYTHON=python2 make lint-ci
```

Sets the `PYTHON` environment variable in the shell and runs `make`, which turns environment variables into `make` variables: https://www.gnu.org/software/make/manual/html_node/Environment.html#Environment

```console
make lint-ci PYTHON=python2
```

Overrides the `make` variable from the command line: https://www.gnu.org/software/make/manual/html_node/Overriding.html